.NET Standard 2 does not support net461 (#29599)
dotnet/docs: includes/net-standard-2.0.md

| .NET implementation         | Version support                                   |
|-----------------------------|---------------------------------------------------|
| .NET and .NET Core          | 2.0, 2.1, 2.2, 3.0, 3.1, 5.0, 6.0                 |

> | .NET Framework <sup>1</sup> | <sup>2</sup> 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8        |

Reviewer: ```suggestion
| .NET Framework <sup>1</sup> | 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8 <sup>2</sup>        |
```

Reviewer 2: I think we should revert this part.

> | .NET Framework <sup>1</sup> | <sup>2</sup> 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8        |

Reviewer 3: ```suggestion
| .NET Framework <sup>1</sup> | 4.6.1 <sup>2</sup>, 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8 |
```

Revert this per @terrajobst

<sup>1</sup> The versions listed for .NET Framework apply to .NET Core 2.0 SDK and later versions of the tooling. Older versions used a different mapping for .NET Standard 1.5 and higher. You can [download tooling for .NET Core tools for Visual Studio 2015](https://github.com/dotnet/core/blob/main/release-notes/download-archives) if you cannot upgrade to Visual Studio 2017 or a later version.

> <sup>2</sup> The versions listed here represent the rules that NuGet uses to determine whether a given .NET Standard library is applicable. For .NET Framework projects that need to use such libraries, we recommend that you upgrade the project to target .NET Framework 4.7.2 or higher.

Reviewer: ```suggestion
<sup>2</sup> The versions listed here represent the rules that NuGet uses to determine whether a given .NET Standard library is applicable. For .NET Framework projects that need to use .NET Standard 2.0, we recommend that you upgrade the project to target .NET Framework 4.7.2 or higher.
```

Reviewer 2: I like that!
